lib/Layouter.js

```diff
@@ -47,6 +47,11 @@ export class Layouter {
 
     const flowElements = root.flowElements;
```

Comment:
Alternative ([null object pattern](https://en.wikipedia.org/wiki/Null_object_pattern)): 

```suggestion
    const flowElements = root.flowElements || [];
```

Reply:
Thanks for the suggestion, resolved.